test/dlc_tests/container_tests/bin/pytorch_tests/testPTHVD

```diff
@@ -6,5 +6,6 @@ LOG_DIR=${HOME_DIR}/logs
 HOROVOD_VERSION=v0.16.4
 
 git clone -b ${HOROVOD_VERSION} https://github.com/horovod/horovod.git ${HOME_DIR}/artifacts/horovod
+pip uninstall -y smdistributed_modelparallel
```

Comment:
why to uninstall it

Reply:
Horovod has been found to break the latest Rubik binary, will send more info.

Reply:
If Horovod breaks the latest SMMP binary, isn't that going to cause issues in the SMMP tests, rather than in the Horovod integ tests?

Reply:
the problem is a bit complicated. So the installation of horovod and smmodelparallel binary + runtime of horovod (e.g. import), would trigger the illegal memory access of some sort. Previously the SMMP test was broken because of a horovod trigger in the smdebug. Thus with the upgraded smdebug, SMMP test is green now. Horovod test now needs fixing since the installation of SMMP is still present while horovod is imported.